### DISCUSSION TITLE
Import error

### DISCUSSION THREAD

**Discussion by wdg118** [zqfang/GSEApy#167] [Q&A]
Hi,

I've installed gseapy. However after searching I cannot find an answer to the error

`ImportError: /lib64/libc.so.6: version 'GLIBC_2.25' not found (required by /home/unix/wgodfrey/.local/lib/python3.8/site-packages/gseapy/gse.cpython-38-x86_64-linux-gnu.so)`

Can anyone please help ?

Thanks,

Will

**Comment by zqfang** [comment]
I think your gcc version is too old. try to install a newer gcc. 

or install rust first, and then compile gseapy by yourself

```shell
pip install gseapy --no-binary :all:
```

**Reply by wdg118** [reply]
I'm unable to get a new gcc version as I do not have sudo privileges. Is there a workaround ?

**Reply by zqfang** [reply]
How about  the command below?  I've update the binaries to PYPI. the new binarires should work for most platforms.

```shell
pip install gseapy --force 
```


I think the easies way to install gseapy is conda. You won't get GLIBC missing. not sudo required 

```shell
conda install -c bioconda gseapy
```
